src/steps/resource-manager/management-groups/index.ts

```diff
@@ -113,7 +113,7 @@ export async function getGraphObjectsForManagementGroup(options: {
           _mapping: {
             sourceEntityKey: managementGroupEntity._key,
             relationshipDirection: RelationshipDirection.FORWARD,
-            targetFilterKeys: [['_type', 'id']],
+            targetFilterKeys: [['id']],
             targetEntity: {
               _type: SubscriptionEntities.SUBSCRIPTION._type,
```

Comment:
Alternatively, I believe you could have explicitly set the `_key` here: 

```suggestion
              _key: child.id,
              _type: SubscriptionEntities.SUBSCRIPTION._type,
```

Reply:
That was my original idea. But Adam and I [concluded that the mapper](https://jptrone.slack.com/archives/C02FM0LLNNA/p1689174366836889?thread_ts=1689150465.567359&cid=C02FM0LLNNA) will not respect the `_key` value and create a new one. Do you know if it would respect the `_key` value given to it?